R/plot.check_model.R

```diff
@@ -10,6 +10,7 @@ plot.see_check_model <- function(x,
   check <- attr(x, "check")
   size_point <- attr(x, "dot_size")
   size_line <- attr(x, "line_size")
+  show_labels <- attr(x, "show_labels") %||% TRUE
```

Comment:
Is `%||%` a base R function, or does it require _magrittr_?

Reply:
I added it as an internal utility function here: https://github.com/easystats/see/blob/034df73816fb747b91d542075b11ff4841278e7c/R/utils.R

Reply:
Oh, I suppose we already import *rlang*, so could just import it from there.